### PR TITLE
Phase3

### PR DESCRIPTION
**Pull request by mishal23**
- Includes dependency management
- Ratings UI
- Download stats

### CODE REVIEW

File: src/appstore/settings/base.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 # from django.core.urlresolvers import reverse_lazy
 from django.urls import reverse_lazy
 from os.path import dirname, join, exists
+import rest_framework
```
Comment (abhijithanilkumar) [inline src/appstore/settings/base.py]: Do we need this import?

File: src/appstore/settings/base.py [hunk 2]
```diff
@@ -83,6 +85,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    # 'django.contrib.sites',
```
Comment (abhijithanilkumar) [inline src/appstore/settings/base.py]: Why is this commented?

File: src/appstore/settings/base.py [hunk 3]
```diff
@@ -128,6 +133,8 @@
     'PAGE_SIZE': 20
 }
 
+# SITE_ID = 1
```
Comment (abhijithanilkumar) [inline src/appstore/settings/base.py]: Is this required?

File: src/download/apps.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class DownloadConfig(AppConfig):
```
Comment (abhijithanilkumar) [inline src/download/apps.py]: Is this required?

File: src/util/parse_bake.py [hunk 5]
```diff
@@ -1,4 +1,6 @@
 import xml.etree.ElementTree as ET
+from django.conf import settings
+# from django.contrib.sites.models import Site
```
Comment (abhijithanilkumar) [inline src/util/parse_bake.py]: Remove?

File: src/appstore/settings/base.py (hunk 1 above)
Comment (mishal23) [reply src/appstore/settings/base.py]: Yes, it's required for setting the Pagination Number.

File: src/appstore/settings/base.py (hunk 2 above)
Comment (mishal23) [reply src/appstore/settings/base.py]: Not needed, was trying out something so added, later forgot to remove, will remove.

File: src/appstore/settings/base.py (hunk 3 above)
Comment (mishal23) [reply src/appstore/settings/base.py]: Not needed, was trying out something so added, later forgot to remove, will remove

File: src/util/parse_bake.py (hunk 5 above)
Comment (mishal23) [reply src/util/parse_bake.py]: Will remove. 👍

File: src/download/apps.py (hunk 4 above)
Comment (mishal23) [reply src/download/apps.py]: It got created on creating the app by Django.

**Comment by abhijithanilkumar** [comment]
Looks good to me! Merging.

**Pull request merged**